openhtf/io/frontend/src/app/index.html

```diff
@@ -0,0 +1,33 @@
+<html>
+  <head>
+    <title>OpenHTF</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">    
+    <link rel="stylesheet" href="/styles.css">
+    <script src="/scripts/vendor.bundle.js"></script>
+    <script defer src="/scripts/app.bundle.js"></script>
+    <script>
+      var FRONTEND_SERVER = '{{host}}:{{port}}';
+    </script>
+  </head>
+  <base href="/">
+
+  <!-- 3. Display the application -->
```

Comment:
Ah, this is the non-prebuilt version. Remove this comment. It would be nice if these pages commented on what their purpose is, since I can't know without either reading/knowing the entire frontend or running it.